File: qqbot-client/src/main/java/com/mieai/qqbot/client/QqMediaUploadResult.java
package com.mieai.qqbot.client;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public record QqMediaUploadResult(
        @JsonProperty("file_uuid") String fileUuid,
        @JsonProperty("file_info") String fileInfo,
        Integer ttl) {
}

File: qqbot-client/src/main/kotlin/com/mieai/qqbot/client/QqMediaUploadResult.kt
package com.mieai.qqbot.client

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class QqMediaUploadResult(
    @JsonProperty("file_uuid") val fileUuid: String?,
    @JsonProperty("file_info") val fileInfo: String?,
    val ttl: Int?,
)
